### PR TITLE
Add Custom Items Script

### PR DESCRIPTION
**Pull request by tdkollins**
This PR adds a script which allows for adding custom items to the game.

### CODE REVIEW

**Comment by tdkollins** [comment]
I guess this is more custom weapons instead of custom items for now.

File: worlds/tits_the_3rd/util.py [hunk 1]
```diff
@@ -121,3 +121,45 @@ def prompt_and_create_backup(backup_directory: str, copy_from_directory: str):
...
+            while fp.tell() < first_item_addr:
+                item_offset = int.from_bytes(fp.read(2), byteorder="little")
+                offsets.append(item_offset)
+            for idx in range(len(offsets)):  # pylint: disable=consider-using-enumerate
+                fp.seek(offsets[idx])
+                item_id = int.from_bytes(fp.read(4), byteorder="little")
+                fp.seek(offsets[idx] + 8)
+                name = b""
+                desc = b""
+                while True:
```
Comment (nnguyen259) [inline worlds/tits_the_3rd/util.py:149]: Can definitely split this off to its own parse_string() method for reusing later

**Comment by nnguyen259** [comment]
> I guess this is more custom weapons instead of custom items for now.

I guess if we want it to be more generic we just need to expand this a little bit. The code is already there for most of the stuffs needed.

**Comment by tdkollins** [comment]
We can make it more generic later when needed, since I dont expect we'll need to inject many custom items other than weapons.

**Pull request merged**